scripts/deploy-docs

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+set -u
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Deploy Swagger spec to a S3 website bucket.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    elif [ "${1:-}" = "--dryrun" ]; then
+        echo "Syncing docs to ${RF_SWAGGER_BUCKET} (dryrun)"
+        aws s3 sync --delete --dryrun \
+                "${DIR}/../docs/swagger" \
```

Comment:
I think here we _only_ want the YML file to be synchronized vs. the entire Swagger UI.